mkosi/__init__.py

```diff
@@ -6056,6 +6056,49 @@ def virt_name(args: CommandLineArguments) -> str:
     return cast(str, name[:13])
 
 
+def has_networkd_vm_vt() -> bool:
+    for path in ["/usr/lib/systemd/network", "/etc/systemd/network"]:
```

Comment:
On non-usr-merged distros this is in /lib, so both should be checked

Reply:
Not that those are really common anymore. debootstrap defaults to merged usr and if my recollection is correct only buildds are currently not installed with merged usr. The crowd opting for non-merged usr might not be the one using mkosi. ;)

Reply:
True, but there's also the case of old systems that were never reinstalled and just upgraded from before the default changed, and didn't manually switch. Like my desktop :-)

Reply:
We already so some similar stuff when building unified kernel images so let's include the same logic here as well